conftest.py

```diff
@@ -12,6 +12,9 @@ def pytest_addoption(parser):
     parser.addoption(
         "--maxerrors", metavar='INT', type=int, help="Maximum number of circuit errors to tolerate (Use with --runnightly)", default=0
```

Comment:
@kkunal1408 Do you know what the `metavar='INT'` argument is for here? Should be get rid of it?

Reply:
I don't think it is used anywhere. Not sure if parijat was planning to use it somewhere.